CHANGELOG.md

```diff
@@ -6,4 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.0.1]
+### Added
+* `fetch_for_scene` function to download the best available orbit information for a scene.
+* Tests for the `fetch_for_scene` function.
+* Support for packaging into a Python Library.
```

Comment:
I think we can omit these, as these release notes will potentially be read by users, and they will only care about the first bullet point.
```suggestion
```